src/mame/mr/dribling.cpp

```diff
@@ -264,20 +293,39 @@ void dribling_state::misc_w(uint8_t data)
 void dribling_state::sound_w(uint8_t data)
 {
 	// bit 7 = stop palla (ball stop)
+	m_i_stop_palla->write_line(BIT(data, 7));
+
 	// bit 6 = contrasto (tackle)
+	m_i_contrasto->write_line(BIT(data, 6));
+
 	// bit 5 = calcio a (kick a)
+	m_i_calcio_a->write_line(BIT(data, 5));
+
 	// bit 4 = fischio (whistle)
+	m_i_fischio->write_line(BIT(data, 4));
+
 	// bit 3 = calcio b (kick b)
+	m_i_calcio_b->write_line(BIT(data, 3));
+
 	// bit 2 = folla a (crowd a)
+	m_i_folla_a->write_line(BIT(data, 2));
+
 	// bit 1 = folla m (crowd m)
+	m_i_folla_m->write_line(BIT(data, 1));
+
 	// bit 0 = folla b (crowd b)
+	m_i_folla_b->write_line(BIT(data, 0));
+	
 	LOGSOUND("%s:sound_w(%02X)\n", machine().describe_context(), data);
 }
 
 
 void dribling_state::pb_w(uint8_t data)
 {
-	// write PB0-7
+	for (int i=0; i < 8; i++)
+	{
+		m_i_pb.at(i)->write_line(BIT(data, i));
+	}
```

Comment:
Same here – don’t use `at(...)` unnecessarily when you can safely index it like an array, and you aren’t planning to catch exceptions anyway.

Reply:
Done